src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -225,6 +231,19 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
     isOpen?: boolean;
   }
 
+  const getManifestName = (uuid: string) => {
+    return data.find((entry) => entry.uuid == uuid)?.name;
+  };
+
+  const handleDeleteManifestConfirmationModalToggle = () => {
+    setIsDeleteManifestConfirmationModalOpen(!isDeleteManifestConfirmationModalOpen);
+  };
+
+  const openDeleteConfirmationModal = (uuid: string) => {
+    setCurrentDeletionUUID(uuid);
+    handleDeleteManifestConfirmationModalToggle();
+  };
+
```

Comment:
So I found a kind of annoying bug, where if you have the allocations list row open on a manifest, and then delete the manifest, it ends up opening the row of the manifest after that. 

I think there's actually a not so bad fix though, bc the index of the currently opened detail panel (from 0 to 9) is already being kept track of in currentDetailRowIndex. So you could do something like

```
const closeDeletedManifestEntitlementsList = () => {
    const newRowExpandedStatus = [...rowExpandedStatus];
    newRowExpandedStatus[currentDetailRowIndex] = false;
    setRowExpandedStatus(newRowExpandedStatus);
  };

  const handleDeleteManifestSuccess = () => {
    closeDeletedManifestEntitlementsList();
    closeDetailsPanel();
  };
```

and then for the delete manifest modal's onSuccess just change that onSuccess prop to be `onSuccess={handleDeleteManifestSuccess}`